backend/lib/models/Comment.js

```diff
@@ -15,6 +16,7 @@ const commentSchema = new Schema(
       ref: "User",
       type: [ObjectId],
     },
+    timeElapsed : String,
```

Comment:
```suggestion
    timeElapsed : String,
```

I don't think it makes sense to store this in the DB.

We can add a virtual to the model with similar logic to post save hook below. FYI we want to return with find & update as well (virtuals are not automatically always returned by default) - see https://stackoverflow.com/a/26909782/4582054.

Reply:
a-ha! thanks! I was looking for a way to dynamically set the computed property without storing it in the DB. Thanks for that info!

Reply:
I had to use  `forEach` https://docs.mongodb.com/manual/reference/method/cursor.forEach/ to set the `timeElapsed` property on GET because you cannot query against a virtual property https://stackoverflow.com/questions/27536383/querying-on-a-virtual-property-in-mongoose